layouts/partials/header.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html>
-    <head>
+    <head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# article: http://ogp.me/ns/article#">
```

Comment:
> can you please provide more context why is this needed (cant tell from the link) ?

This error appeared in Facebook debugger when prefix not set.

![image](https://cloud.githubusercontent.com/assets/4950792/21558931/d21706ec-ce85-11e6-8e53-4c714bb867ef.png)

> we are including open graph stuff only when opengraph = true param is enabled (SEE 

oh...sorry. I missed it.
How to set OGP was difficult for me to understand.
It becomes easy to understand if you add it to default.md